src/main/kotlin/no/nav/helse/soknad/BarnDetaljer.kt

```diff
@@ -69,7 +69,7 @@ internal fun BarnDetaljer.validate(): MutableSet<Violation> {
                 parameterName = "barn.fødselsnummer",
                 parameterType = ParameterType.ENTITY,
                 reason = "Ikke gyldig fødselsnummer.",
-                invalidValue = fødselsnummer
+                invalidValue = "${fødselsnummer!!.take(6)}*****"
```

Comment:
fødselsnummer kan vel være null her. 
Prøv heller `"${fødselsnummer?.take(6)}*****"`

Reply:
Det er i en if (fødselsnummer != null) så burde ikke. Er pga "var" at compiler klager. Men kan endre

Reply:
Dette kan forresten resultere i "null*****". 
Bedre om det skrives slik: 
`fødselsnummer ?.let {"${it.take()6}*****"}`

Reply:
Endet til ? :)